### DISCUSSION TITLE
Any plan to support Typesense soon?

### DISCUSSION THREAD

**Discussion by tungle** [requarks/wiki#5472] [Help / Questions] [answered]
I'm running an instance of [Typesense](https://typesense.org/) server, would appreciate if Typsense is added to the list of supported search engine. Typsense is so far the best free search engine for internal documents in my experience.

**Comment by NGPixel** [comment, marked as answer]
There're no plans to add additional search engines to v2 at this point and v3 will not support external search engines.
